.circleci/config.yml

```diff
@@ -0,0 +1,138 @@
+# Immuni CircleCI configuration.
+# Copyright (C) 2020 Presidenza del Consiglio dei Ministri.
+# Please refer to the AUTHORS file for more information.
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+version: 2.1
+description: Maintain consistent security and quality standards.
+
+# TODO: Evaluate promotion to orbs.
+commands:
+  poetry-install:
+    description: |
+      Install Poetry and the Python dependencies from the poetry.lock with Poetry.
+      The virtualenv containing the installed packages will automatically be cached and restored.
+    parameters:
+      additional_parameters:
+        description: Additional parameters to append to the poetry install command (e.g., --extras)
+        type: string
+        default: ""
+
+      deps_cache_version:
+        description: Dependencies cache version identifier. Bump this to clear the virtualenv cache.
+        type: string
+        default: "v1"
+
+    steps:
+      - run:
+          name: "[python] Setup Poetry environment"
+          command: echo 'export POETRY_HOME=./.poetry; export PATH=$POETRY_HOME/bin:$PATH' >> $BASH_ENV
+      - restore_cache:
+          name: "[python] Restore the Poetry cache"
+          keys:
+            - poetry-754dbf80dc022b89974288cff10b40ab2f1c2697
```

Comment:
Couldn't we use the poetry version here instead of the SHA?

Reply:
I used the same used in the Dockerfile to keep consistency. IMHO, this would seldomly change.

Do you think it's fine?